pytest_fixtures/component/provision_gce.py

```diff
@@ -26,6 +27,21 @@ def gce_cert(default_sat):
     return cert
 
 
+@pytest.fixture(scope='session')
+def gce_cert_puppet(session_puppet_enabled_sat):
+    _, gce_cert_file = mkstemp(suffix='.json')
+    cert = json.loads(settings.gce.cert)
+    cert['local_path'] = gce_cert_file
+    with open(gce_cert_file, 'w') as f:
+        json.dump(cert, f)
+    session_puppet_enabled_sat.put(gce_cert_file, settings.gce.cert_path)
+    if session_puppet_enabled_sat.execute(f'[ -f {settings.gce.cert_path} ]').status != 0:
+        raise GCECertNotFoundError(
```

Comment:
I think it would be better to call `pytest.fail` here to explicitly fail the setup.